www/templates/menu.html

```diff
@@ -34,6 +34,10 @@ <h1 class="title">PVTA</h1>
           <i class="icon ion-ios-location" style="margin-right: 10px"></i>
           Stops
         </ion-item>
+        <ion-item menu-close href="#/app/map">
+          <i class="icon ion-android-map" style="margin-right: 10px"></i>
```

Comment:
Not saying I'm against introducing a Map option to the sidebar, but I'm curious as to what uses it'd have.

So, a user selects the Map from the sidebar and is brought to...well, a map, centered on their location.  That's it though, right?  They still have to go through another page to see anything else on the map.